### ISSUE TITLE
Syntax error parsing commented-out graphql code in schema

### ISSUE THREAD

**Issue by clintfoster** [aws-amplify/amplify-category-api#1157]
### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

v16.13.1

### Amplify CLI Version

10.6.1

### What operating system are you using?

Mac

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

No manual changes made

### Describe the bug

Commented-out code in `schema.graphql` causes syntax error:

>Syntax Error: mismatched brackets found in the schema. Unexpected ) at line 3 in the schema.

### Expected behavior

Commented-out graphql code should be ignored.

### Reproduction steps

1. `mkdir comment`
1. `cd comment`
1. `amplify init` (accept defaults)
1. `amplify add api` (graphql, blank schema, edit y, insert schema below)
1. `amplify push`

Error:

>"Syntax Error: mismatched brackets found in the schema. Unexpected ) at line 3 in the schema.""

# Schema

```graphql
type Usr @model {
  id: ID! @primaryKey
  #  orgs: [Org] @manyToMany(relationName: "OrgUsrJn")
}
```

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Additional information

I believe this started happening after one of the recent updates to the amplify CLI since I have not made significant schema changes.

This problem is resolved by removing the parenthetical portion of the commented-out code:

```graphql
(relationName: "OrgUsrJn")
```

In other words, the following schema with the above removed from the commented-out line works:

```graphql
type Usr @model {
  id: ID! @primaryKey
  #  orgs: [Org] @manyToMany
}
```

### Before submitting, please confirm:

- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

**Comment by josefaidt** [comment]
Hey @clintfoster :wave: thanks for raising this! This appears to be a duplicate of https://github.com/aws-amplify/amplify-category-api/issues/1143, I'll close this issue in favor of tracking the existing item. Please subscribe to #1143 for further updates
